package.json

```diff
@@ -88,15 +92,19 @@
     "files": [
       {
         "path": "./dist/onfido.min.js",
-        "maxSize": "258 kB"
+        "maxSize": "270 kB"
       },
       {
         "path": "./dist/onfido.crossDevice.min.js",
-        "maxSize": "1.3 kB"
+        "maxSize": "70 kB"
       },
       {
         "path": "./dist/style.css",
         "maxSize": "100 kB"
+      },
+      {
+        "path": "./dist/onfido.crossDevice.css",
+        "maxSize": "2 kB"
```

Comment:
So this gets dynamically loaded too when I import the crossDevice component? What if I import the SDK from the dist?
```
require('onfido-sdk-ui/dist/style.css')
const { init } = import('onfido-sdk-ui/dist/onfido.min.js')